orchestrator-salt/src/main/resources/salt/salt/postgresql/scripts/backup_db.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+if [ $# -ne 6 ]; then
+  echo "Invalid inputs provided"
+  echo "Script accepts five inputs:"
+  echo "  1. Cloud Provider (azure | aws)"
+  echo "  2. PostgreSQL host name."
+  echo "  3. PostgreSQL port."
+  echo "  4. Object Storage Service url to place backups."
+  echo "  5. PostgreSQL user name."
+  echo "  6. PostgreSQL password."
+  exit 1
+fi
+
+CLOUD_PROVIDER="$1"
+HOST="$2"
+PORT="$3"
+BACKUP_LOCATION="$4"
+USERNAME="$5"
+export PGPASSWORD="$6" # We can provide the password to pg_dump through this variable, or in ~/.pgpass
```

Comment:
If you're exporting this, you might want to unset it at the end of the script. I don't actually think it'll live outside the script's context, but just in case.

Reply:
I think this should really live in the `.pgpass` file for proper security. I'll think 
 (talk) about whether we need to that now or later.

Reply:
Postgres will attempt to read the `~/.pgpass` file if it's present. So we should see about adding that to Cloudbreak.